adobe-acrobat-sign-connector-product/README.md

```diff
@@ -7,25 +7,6 @@ The Adobe Acrobat Sign Connector is a project that simplifies the authentication
 
 An Adobe Sign account needs to be created to setup and use the connector.
```

Comment:
I think this sentence should be the first one after ## Adobe Sign account creation